cypress/src/tests/e2eTests/verifyZeroTotalCheckoutWithGiftCardAndGiftOptions.spec.js

```diff
@@ -34,7 +34,7 @@ import {
   customerShippingAddress
 } from '../../fixtures/index';
 
-describe('Verify price summary on cart', () => {
+describe('Verify price summary on cart',{ tags: '@paas' }, () => {
```

Comment:
I guess you added these tags to identify when we should run a test or not. I would expect the default is that we run on both paas and saas, and then if absolutelty necessary we skip or add a specific tag for the env where it does work

Reply:
Agree, let me update but there can be nested cases in future. I will start with generic @skip tag and we might end up saas/paas specific skip tags for some tests.